src/Liquidator.sol

```diff
@@ -71,28 +71,28 @@ contract Liquidator is Ownable {
     /**
      * @notice Sets the factory address on the liquidator.
      * @dev The factory is used to fetch the isVault bool in elevated().
-     * @param _factory the factory address.
+     * @param factory the factory address.
      */
-    function setFactory(address _factory) external onlyOwner {
-        factoryAddress = _factory;
+    function setFactory(address factory) external onlyOwner {
+        factoryAddress = factory;
     }
 
     /**
      * @notice Sets the protocol treasury address on the liquidator.
      * @dev The protocol treasury is used to receive liquidation rewards.
-     * @param _protocolTreasury the protocol treasury.
+     * @param protocolTreasury_ the protocol treasury.
      */
-    function setProtocolTreasury(address _protocolTreasury) external onlyOwner {
-        protocolTreasury = _protocolTreasury;
+    function setProtocolTreasury(address protocolTreasury_) external onlyOwner {
+        protocolTreasury = protocolTreasury_;
```

Comment:
maybe to refactor this to 
```suggestion
        protocolTreasuryAddress = protocolTreasury;
```
what do you think @Thomas-Smets @j-vp

Reply:
or change `factoryAddress` to `factory` in the contract for consistent naming

Reply:
Jep, the shorter the name the better.
Also bad practice to repeat the type in the name